examples/openthread/main.c

```diff
@@ -16,15 +16,37 @@
 #include <stdio.h>
 
 #include "ot.h"
+#include "openthread/thread.h"
+
+static void _panid_handler(event_t *event);
+
+static event_t event_panid = {
+    .handler = _panid_handler
+};
+
+static void _panid_handler(event_t *event)
+{
+    (void) event;
+
+    /* It's safe here to call any OpenThread specific API, since this code runs
+     * in the same thread as the OpenThread tasklet handler */
+
+    /* Get PanID from OpenThread API */
+    uint16_t panid = otLinkGetPanId(openthread_get_instance());
+    printf("Current panid: 0x%x\n", panid);
+}
 
 int main(void)
 {
-    puts("This a test for OpenThread");
     /* Example of how to call OpenThread stack functions */
-    puts("Get PANID ");
-    uint16_t panid = 0;
-    uint8_t res = ot_call_command("panid", NULL, (void*)&panid);
-    printf("Current panid: 0x%x (res:%x)\n", panid, res);
+    puts("This a test for OpenThread");
```

Comment:
openthread is in examples/ so not really a test. I know it was there already but maybe we can take the opportunity of this PR to adapt.

Reply:
Sure, I can do it

Reply:
done!